hztb-servicemanager/src/main/java/com/svs/hztb/service/impl/GroupDataServiceImpl.java

```diff
@@ -33,72 +32,81 @@
 
 	@Autowired
 	GroupRepository groupRepository;
-	
+
 	@Autowired
 	UserGroupEntityRepository userGroupEntityRepository;
-	
+
 	@Autowired
 	UserRepository userRepository;
-	
+
 	@Override
 	public GroupOutput removeGroup(GroupInput groupInput) {
-		
+
 		GroupOutput groupOutput = null;
 		try {
 			groupRepository.setGroupNameFor(DEFAULT_GROUP_NAME, groupInput.getGroupId(), groupInput.getUserId());
 			groupOutput = buildGroupOutput();
-		} catch(Exception e) {
-			
+		} catch (Exception e) {
+
 		}
 		return groupOutput;
-		
+
 	}
-	
+
 	@Override
 	public GroupOutput createGroup(GroupInput groupInput) {
 		GroupOutput groupOutput = null;
 		try {
-			//Create group
+			// Create group
 			GroupEntity groupEntity = FunctionUtils.convert(groupInput, new GroupInputToEntityConverter());
 			groupRepository.saveAndFlush(groupEntity);
-		//	groupRepository.findByGroupName(groupEntity.getGroupName());
-		
-			//Add group members
+			// groupRepository.findByGroupName(groupEntity.getGroupName());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)